### PR TITLE
French translation for KEYCLOAK-2120

### PR DESCRIPTION
**Pull request by gaetancollaud**
* Added translation for KEYCLOAK-2120
* Removed link in translation since apps are now in a list (and html was escaped)

### CODE REVIEW

**Review by jayblanc — APPROVED** [review]

**Review by stianst — APPROVED** [review]

File: themes/src/main/resources-community/theme/base/account/messages/messages_fr.properties [hunk 1]
```diff
@@ -99,10 +99,24 @@ revoke=R\u00e9voquer un droit
 
 configureAuthenticators=Authentifications configur\u00e9es.
 mobile=T\u00e9l\u00e9phone mobile
-totpStep1=Installez <a href="https://freeotp.github.io/" target="_blank">FreeOTP</a> ou bien Google Authenticator sur votre mobile. Ces deux applications sont disponibles sur <a href="https://play.google.com">Google Play</a> et Apple App Store.
+totpStep1=Installez une des applications suivantes sur votre mobile
```
Comment (sebastienblanc) [inline themes/src/main/resources-community/theme/base/account/messages/messages_fr.properties]: Now we miss the links, it needs to be escaped like here https://github.com/keycloak/keycloak/blob/master/themes/src/main/resources-community/theme/base/account/messages/messages_es.properties#L98

File: themes/src/main/resources-community/theme/base/account/messages/messages_fr.properties [hunk 2]
```diff
@@ -99,10 +99,24 @@ revoke=R\u00e9voquer un droit
...
 totpStep2=Ouvrez l''application et scannez le code-barres ou entrez la clef.
 totpStep3=Entrez le code \u00e0 usage unique fourni par l''application et cliquez sur Sauvegarder pour terminer.
 
+totpManualStep2=Ouvrez l''application et entrez la clef
+totpManualStep3=Utilisez les valeurs de configuration suivante si l''application les authorise
+totpUnableToScan=Impossible de scanner ?
+totpScanBarcode=Scanner le code bare?
```
Comment (sebastienblanc) [inline themes/src/main/resources-community/theme/base/account/messages/messages_fr.properties]: `Scanner le code barre ?`

File: themes/src/main/resources-community/theme/base/account/messages/messages_fr.properties [hunk 3]
```diff
@@ -99,10 +99,24 @@ revoke=R\u00e9voquer un droit
...
+
+totp.totp=Bas\u00e9 sur le temps
+totp.hotp=Bas\u00e9 sur un compteur
+
+totpType=Type
+totpAlgorithm=Algorithme
+totpDigits=Digits
+totpInterval=Intervale
```
Comment (sebastienblanc) [inline themes/src/main/resources-community/theme/base/account/messages/messages_fr.properties]: `Intervalle`

File: themes/src/main/resources-community/theme/base/account/messages/messages_fr.properties [hunk 4]
```diff
@@ -99,10 +99,24 @@ revoke=R\u00e9voquer un droit
...
+totpManualStep3=Utilisez les valeurs de configuration suivante si l''application les authorise
+totpUnableToScan=Impossible de scanner ?
+totpScanBarcode=Scanner le code bare?
+
+totp.totp=Bas\u00e9 sur le temps
+totp.hotp=Bas\u00e9 sur un compteur
+
+totpType=Type
+totpAlgorithm=Algorithme
+totpDigits=Digits
```
Comment (sebastienblanc) [inline themes/src/main/resources-community/theme/base/account/messages/messages_fr.properties]: `Chiffres`

File: themes/src/main/resources-community/theme/base/account/messages/messages_fr.properties [hunk 5]
```diff
@@ -99,10 +99,24 @@ revoke=R\u00e9voquer un droit
 
 configureAuthenticators=Authentifications configur\u00e9es.
 mobile=T\u00e9l\u00e9phone mobile
-totpStep1=Installez <a href="https://freeotp.github.io/" target="_blank">FreeOTP</a> ou bien Google Authenticator sur votre mobile. Ces deux applications sont disponibles sur <a href="https://play.google.com">Google Play</a> et Apple App Store.
+totpStep1=Installez une des applications suivantes sur votre mobile
 totpStep2=Ouvrez l''application et scannez le code-barres ou entrez la clef.
 totpStep3=Entrez le code \u00e0 usage unique fourni par l''application et cliquez sur Sauvegarder pour terminer.
 
+totpManualStep2=Ouvrez l''application et entrez la clef
+totpManualStep3=Utilisez les valeurs de configuration suivante si l''application les authorise
```
Comment (sebastienblanc) [inline themes/src/main/resources-community/theme/base/account/messages/messages_fr.properties]: `autorise`

**Comment by sebastienblanc** [comment]
Ok this was already merged ;) , I will provide another PR that fixes some typos.

**Comment by jayblanc** [comment]
Sorry for missing that... even reading 5 times !

**Comment by gaetancollaud** [comment]
@sebastienblanc  Outch, shame on me. Sorry for all those dummy mistakes !

**Comment by sebastienblanc** [comment]
You can review the new PR here https://github.com/keycloak/keycloak/pull/5045

File: themes/src/main/resources-community/theme/base/account/messages/messages_fr.properties (hunk 5 above)
Comment (ruchg1337) [reply themes/src/main/resources-community/theme/base/account/messages/messages_fr.properties]: clé is more common now than clef

**Comment by gaetancollaud** [comment]
I did. Having so many mistakes made me check the Larousse. `code-barres` ends with an `s`. I wasn't aware of it.

**Pull request merged**